WikiRacer.py

```diff
@@ -24,6 +33,9 @@ def parse(argv):
     parser.add_argument("--quiet", "-q", help="Create fewer log messages", action="count")
     parser.add_argument("--no-console", help="Disable console logging", action="store_true")
     parser.add_argument("--no-file", help="Disable file logging", action="store_true")
+    parser.add_argument("--api", help="The api to use. One of {}".format(getValidAPIs()), default="WikipediaApi")
```

Comment:
Apparently there's a `choices` argument that could be useful here and in the heuristic arg: https://docs.python.org/3/library/argparse.html#choices